dev/doc/profiling.md

```diff
@@ -70,21 +76,42 @@ perf script --time '0%-100%'  |
 EOF
         ) |
         flamegraph.pl
-~~~~~
+```
 
-## Memory
+## Memory (memtrace)
+
+[memtrace](https://github.com/janestreet/memtrace) is a client library
+for OCaml's Memprof statistical memory profiler.
+
+See this blog post for more details:
+https://blog.janestreet.com/finding-memory-leaks-with-memtrace/
+
+To profile a file, we recommend you make a copy (to avoid issues with
+artifacts), then you can do:
+
+```
+dune build theories/Strings/Byte.vo  # to build deps of Byte
+MEMTRACE=trace-byte.tcr dune exec -- dev/shim/coqc MyByte.v
+memtrace-viewer trace-byte.tcr
+```
+
+## Memory (legacy)
```

Comment:
I agree that this section should go away.

Reply:
Done thanks, let's see how it looks now.